### ISSUE TITLE
ALMNH ES - 6. Taxonomy (Classifications)

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/data-migration#222]
Read the [Taxonomy](http://handbook.arctosdb.org/documentation/taxonomy.html) documentation

For any names that were added in the previous data migration step, there will not be a related classification in any of the local Arctos taxonomy sources. For a small number of new names, these can be created manually by following the instructions in [How To Create Taxa](http://handbook.arctosdb.org/how_to/How-to-Create-Taxa.html). To create a large number of classifications, use the [Bulkload Taxonomy Classifications Tool](http://arctos.database.museum/tools/BulkloadClassification.cfm)

**Comment by Jegelewicz** [comment]
@mbprondzinski I have invited @aklompma to be a member of the Arctos GitHub team and until he joins, I can't assign this to him, but before I give him access to taxonomy, I want to make sure that he has read and understands the above. Please let him know that he should accept the GitHub invite so that he can participate in our conversations!

**Comment by mbprondzinski** [comment]
Yes, I already told him that.  He seemed to think he was a member, but obviously not.  He’s home sick today, but I’ll make sure he accepts the invite!

**Comment by Jegelewicz** [comment]
Bummer - hope is well soon and thanks!

**Comment by aklompma** [comment]
Just working from home while recovering. I did accept an invite recently so not sure what is meant.


I am a member of "Arctos DB organization" according to my profile and part of the following teams "Arctos Users" and "Taxonomy".


Best wishes,

Adiel


________________________________

**Comment by Jegelewicz** [comment]
@aklompma OK - not sure why GitHub wasn't finding you. I've now assigned this issue to you. Once you let me know that you understand the implications of the shared data table that is taxonomy, I'll give you access in Arctos.

Hope you are well soon!
